### PR TITLE
delete third party service

### PR DESCRIPTION
**Pull request by GLYASAI**
删除组件的同时，删除相关的 k8s 对象，包括：deployment，statefulset，ingress，service，endpoint。
避免组件删除后，还有残留的，游离的  k8s 对象。

### CODE REVIEW

File: worker/gc/gc.go [hunk 1]
```diff
@@ -90,3 +90,35 @@ func (g *GarbageCollector) DelPvPvcByServiceID(serviceGCReq model.ServiceGCTaskB
 		logrus.Warningf("service id: %s; delete a collection fo PVC: %v", serviceGCReq.ServiceID, err)
 	}
 }
+
+// DelKubernetesObjects deletes all kubernetes objects.
+func (g *GarbageCollector) DelKubernetesObjects(serviceGCReq model.ServiceGCTaskBody) {
+	deleteOpts := &metav1.DeleteOptions{}
+	listOpts := metav1.ListOptions{
+		LabelSelector: fmt.Sprintf("service_id=%s", serviceGCReq.ServiceID),
```
Comment (barnettZQG) [inline worker/gc/gc.go:98]: LabelSelector should include `creator=Rainbond`?

Comment (GLYASAI) [reply worker/gc/gc.go]: Added

File: worker/gc/gc.go [hunk 2]
```diff
@@ -90,3 +89,43 @@ func (g *GarbageCollector) DelPvPvcByServiceID(serviceGCReq model.ServiceGCTaskB
 		logrus.Warningf("service id: %s; delete a collection fo PVC: %v", serviceGCReq.ServiceID, err)
 	}
 }
+
+// DelKubernetesObjects deletes all kubernetes objects.
+func (g *GarbageCollector) DelKubernetesObjects(serviceGCReq model.ServiceGCTaskBody) {
```
Comment (barnettZQG) [inline worker/gc/gc.go:94]: not delete `configmap` and other resources?

delete endpoint after delete service.

Comment (GLYASAI) [reply worker/gc/gc.go:94]: Added configmaps, secrets and hpa, there should be nothing more

**Review by barnettZQG — APPROVED** [review]
Reviewed

**Pull request merged**